content/agent/basic_agent_usage/amazonlinux.md

```diff
@@ -2,86 +2,152 @@
+
+The Python interpreter embedded with the Agent is located here:  
+`/opt/datadog-agent/embedded/bin/python`.
+The agent also comes with pip, install python libraries using:
 ```
-To receive information about the Agent’s state:
-```shell
-sudo /etc/init.d/datadog-agent info
+sudo -u dd-agent -- /opt/datadog-agent/embedded/bin/pip install <package_name>
 ```
 
-Tracebacks for errors can be retrieved by setting the `-v` flag: (since 3.8.0)
-```shell
-sudo /etc/init.d/datadog-agent info -v
-```
+## Switch between Agent v5 and v6
+### Upgrade to Agent 6
+
+A script is available to automatically install or upgrade the new Agent. It sets up the repos and install the package for you; in case of upgrade, the import tool also searches for an existing `datadog.conf` from a prior version and converts Agent and checks configurations according to the new file format and filesystem layout.
+#### One-step install
+##### To Upgrade
+
+In case you have an Agent version 5.17 or later and you want to import the
+existing configuration:
 
-More information about the metrics, events, and service checks for an [integrations](/integrations) can be retrieved with the check command:
 ```shell
-sudo /etc/init.d/datadog-agent check [integration]
+ DD_UPGRADE=true bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
 ```
 
-Add the check_rate argument to get the most recent values for rates:
+**Note:** the import process won't automatically move custom checks, this is by
+design since we cannot guarantee full backwards compatibility out of the box.
+
+##### To Install Fresh
+
+To install on a clean box (or have an existing agent 5 install from which you do not wish to import the configuration) provide an api key:
+
 ```shell
-sudo /etc/init.d/datadog-agent check [integration] check_rate
+ DD_API_KEY=YOUR_API_KEY bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
 ```
 
-## Configuration
-The configuration file for the Agent is located at `/etc/dd-agent/datadog.conf`
+#### Manual install
+1. Set up apt so it can download through https
 
-Configuration files for integrations are located in `/etc/dd-agent/conf.d/
+    ```shell
+    sudo apt-get update
+    sudo apt-get install apt-transport-https
+    ```
 
-## Troubleshooting
+2. Add the beta repo to your system and import the datadog gpg key
 
-Try running the [info command](#status-and-information) to see the state of the Agent.
-Logs for the subsystems are in the following files:
+    ```shell
+    echo 'deb https://apt.datadoghq.com/ beta main' | sudo tee /etc/apt/sources.list.d/datadog-beta.list
+    sudo apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 C7A7DA52
+    ```
 
-* `/var/log/supervisor/datadog-supervisord.log (since 3.8.0)`
-* `/var/log/datadog/collector.log`
-* `/var/log/datadog/dogstatsd.log`
-* `/var/log/datadog/forwarder.log`
+3. Update Apt and Install the agent
```

Comment:
APT or apt, not Apt.  

Also again lets pick a single case / capitalization format.

Reply:
let's take APT

Reply:
👌